userguide/system.rst

```diff
@@ -146,54 +145,55 @@ can be configured using the General tab:
-   | Console Keyboard Map | drop-down menu | select the keyboard layout                                                                                               |
+   | Console Keyboard Map | drop-down menu | Select a keyboard layout.                                                                                                |
    |                      |                |                                                                                                                          |
    +----------------------+----------------+--------------------------------------------------------------------------------------------------------------------------+
-   | Timezone             | drop-down menu | select the timezone from the drop-down menu                                                                              |
+   | Timezone             | drop-down menu | Select a timezone.                                                                                                       |
    |                      |                |                                                                                                                          |
    +----------------------+----------------+--------------------------------------------------------------------------------------------------------------------------+
-   | Syslog level         | drop-down menu | when :guilabel:`Syslog server` is defined, only logs matching this level are sent                                        |
+   | Syslog level         | drop-down menu | When :guilabel:`Syslog server` is defined, only logs matching this level are sent.                                       |
    |                      |                |                                                                                                                          |
    +----------------------+----------------+--------------------------------------------------------------------------------------------------------------------------+
-   | Syslog server        | string         | *IP address_or_hostname:optional_port_number* of remote syslog server to send logs to; once set, log entries             |
-   |                      |                | are written to both the console and the remote server                                                                    |
+   | Syslog server        | string         | Define an *IP address_or_hostname:optional_port_number* to send logs to. Once set, log entries                           |
+   |                      |                | are written to both the console and the remote server.                                                                   |
    |                      |                |                                                                                                                          |
```

Comment:
*IP address_or_hostname:optional_port_number* should be
**IP address** or **hostname[:port number]**
The choices are to set an IP Address or hostname (with an optional port number after a colon)